horreum-web/src/domain/admin/Datastores.tsx

```diff
@@ -51,73 +59,23 @@ const DatastoresTable = ( props: dataStoreTableProps) => {
 
     };
 
-    const defaultActions = (datastore: Datastore): IAction[] => [
+    const defaultActions = (selectedDatastore: Datastore): IAction[] => [
         {
-            title: `Edit`, onClick: () => editModalToggle(datastore.id)
+            title: `Edit`, onClick: () => props.modifyDatastore(selectedDatastore)
         },
         {
-            title: `Test`, onClick: () => verifyModalToggle(datastore.id)
+            title: `Test`, onClick: () => props.verifyDatastore(selectedDatastore)
         },
         {
             isSeparator: true
         },
         {
-            title: `Delete`, onClick: () => deleteModalToggle(datastore.id)
+            title: `Delete`, onClick: () => props.deleteDatastore(selectedDatastore)
         },
 
     ];
-    const newBackendConfig: ElasticsearchDatastoreConfig | CollectorApiDatastoreConfig = {
-        url: "",
-        apiKey: "",
-        builtIn: false
-    }
-
-    const newDataStore: Datastore = {
-        id: -1,
-        name: "",
-        owner: props.team.key,
-        builtIn: false,
-        access: Access.Private,
-        config: newBackendConfig,
-        type: DatastoreTypeEnum.Postgres
-    }
-
-    const [deleteModalOpen, setDeleteModalOpen] = useState(false);
-    const [editModalOpen, setEditModalOpen] = useState<boolean>(false);
-    const [verifyModalOpen, setVerifyModalOpen] = useState(false);
-    const [datastore, setDatastore] = useState<Datastore >(newDataStore)
-
-    const findDatastore = (id: number) => {
-        return props.datastores.filter( datastore => datastore.id === id).pop() || newDataStore
-    }
-
-    const updateDatastore = ( datastore: Datastore) : void => {
-        setDatastore(datastore)
-    }
-
-    const deleteModalToggle = (id: number) => {
-        setDatastore(findDatastore(id))
-        setDeleteModalOpen(!deleteModalOpen);
-    };
-    const editModalToggle = (id: number) => {
-        setDatastore(findDatastore(id))
-        setEditModalOpen(!editModalOpen);
-    };
-    const verifyModalToggle = (id: number) => {
-        setDatastore(findDatastore(id))
-        setVerifyModalOpen(!verifyModalOpen);
-    };
-
-    const newDatastore = () => {
-        setDatastore(newDataStore)
-        setEditModalOpen(!editModalOpen);
-    };
-
 
     return (
-
-        <Stack hasGutter>
-            <StackItem>
```

Comment:
fix indentation level

Reply:
This refers to a line that has been removed, but I am applied auto formatting to the entire file